docs/node/run-your-node/prerequisites/stake-requirements.md

```diff
@@ -1,38 +1,49 @@
 # Stake Requirements
 
-This page provides an overview of the stake requirements to become a validator
-on the consensus layer, to run a ParaTime compute nodes and to run observer
-nodes on the Oasis Network.
+This page provides the current stake requirements to first register your account
+as an entity on the consensus layer, to become a validator and finally to run a
+ParaTime or the observer compute node.
 
 ## Registering an Entity
 
 To register an entity you are going to need at least: 
 
 |                             | Mainnet  | Testnet  |
-| --------------------------- | -------- | -------- |
+|-----------------------------|----------|----------|
 | Entity Registration         | 100 ROSE | 100 TEST |
-| Each Node Registration      | 100 ROSE | 100 TEST |
 
+You can also fetch the latest network values above by using the
+[`oasis network show native-token`] command in the Oasis CLI.
 
-## Validator Node
+:::tip
 
-To become a validator on the consensus layer, you will need to have enough ROSE
-for **registering your entity**, **registering your validator node**, and be among
-the top entities by stake to be included in the **validator set**.
+To check the balance of your account, you can use one of the blockchain
+explorers like the [Oasis Scan] or the Oasis CLI. If latter, then run the
+[`oasis account show`] command and pass the `--show-delegations` flag to see any
+staked assets.
 
-You can check the current top validators on the blockchain explorers like the [Oasis Scan].
+:::
 
-For the Testnet, you can receive a limited number of TEST tokens by using our [Oasis Network Testnet Faucet][faucet-testnet].
-For more tokens please contact us on our official [Discord in the #testnet channel][discord].
+## Registering a Node
 
-:::tip
+To become a validator on the consensus layer, you will need to have enough ROSE
+staked on your entity account for **registering your node(s)**, and be among the
+top entities by stake to be included in the **validator set**. You will also
+need to register a node, if you want to run an **observer node**.
 
-You can use the Oasis CLI to check your account and the network parameters. Use
-the [`oasis account show`] command to check your entity's account and
-[`oasis network show native-token`] to see the current values used by the network.
+|                           | Mainnet  | Testnet  |
+|---------------------------|----------|----------|
+| Each node registration    | 100 ROSE | 100 TEST |
+| Size of the validator set | 120      | 110      |
```

Comment:
Don't you need +100 tokens to register an entity? I would also remove the validator set size from this table, or make another table or sth for validator set size.

Reply:
I revamped the whole chapter. Please take a look.